src/gluonts/model/forecast.py

```diff
@@ -404,7 +404,7 @@ def alpha_for_percentile(p):
             ptile = percentiles_sorted[i]
             alpha = alpha_for_percentile(ptile)
             plt.fill_between(
-                self.index,
+                self.index.to_timestamp(),
```

Comment:
Should this be done once and for all? If `DatetimeIndex` is what is really needed here?

Reply:
Maybe, but I think we want to replace this entire thing in `0.11` anyway.